templates/newsletter-form.html

```diff
@@ -0,0 +1,62 @@
+  <div class="p-card">
+    <header>
+      <form action="https://pages.canonical.com/index.php/leadCapture/save" method="post" id="mktoForm_1212" novalidate="novalidate">
```

Comment:
This form is closed by the closing header. The form should wrap the entire content of the card.